src/main/java/com/programmers/console/CommandLineApplication.java

```diff
@@ -1,40 +1,92 @@
 package com.programmers.console;
 
-import com.programmers.console.util.Menu;
-import com.programmers.console.util.CommandType;
+import com.programmers.console.util.Command;
 import com.programmers.console.view.Console;
+import com.programmers.voucher.controller.VoucherController;
+import com.programmers.voucher.domain.Discount;
+import com.programmers.voucher.domain.VoucherType;
+import com.programmers.voucher.dto.VoucherRequestDto;
+import com.programmers.voucher.dto.VoucherResponseDto;
 import org.springframework.stereotype.Component;
 
+import java.util.UUID;
+
 @Component
 public class CommandLineApplication {
+
+    private static final String WRONG_INPUT_MESSAGE_FOR_VALUE = "[ERROR] 숫자를 입력해 주세요.";
+
+    private final VoucherController voucherController;
+    private final Console console;
+
     private boolean isRunning = true;
-    private Console console = new Console();
+
+    public CommandLineApplication(Console console, VoucherController voucherController) {
+        this.console = console;
+        this.voucherController = voucherController;
+    }
 
     public void run() {
         while (isRunning) {
             try {
-                console.printMenu();
-                CommandType command = getCommand();
+                Command command = Command.of(console.inputMenu());
```

Comment:
한 줄이 하나의 행위만 하는 것이 좋을 것 같아요!
```suggestion
                String inputMenuText = console.inputMenu()
                Command command = Command.of(inputMenuText);
```

Reply:
이러니 뭔가 어색해보이는 거 같은 데 어떠세요?